splinepy/helpme/create.py

```
@@ -298,6 +297,426 @@ def revolved(
     return type(spline)(**spline_dict)
 
 
+def swept(
```

Ruff (0.15.10)

[warning] Too many branches (40 > 12)

(PLR0912)